Yosemite/Yosemite/Stores/ProductAttributeTermStore.swift

```diff
@@ -61,17 +57,18 @@ private extension ProductAttributeTermStore {
                                          attributeID: attributeID,
                                          pageNumber: fromPageNumber,
                                          pageSize: pageSizeRequest) { [weak self] result in
-            guard let self = self else { return }
+            guard let self else { return }
             switch result {
 
             // If terms count is less than the requested page size, end the recursion and call `onCompletion`
             case let .success(terms) where terms.count < self.pageSizeRequest:
-                self.deleteStaleTerms(siteID: siteID, attributeID: attributeID, activeTerms: synchronizedTerms + terms)
-                onCompletion(nil)
+                deleteStaleTerms(siteID: siteID, attributeID: attributeID, activeTerms: synchronizedTerms + terms) {
+                    onCompletion(nil)
+                }
 
             // If there could be more(non-empty) terms, request the next page recursively.
             case let .success(terms):
-                self.synchronizeAllProductAttributeTerms(siteID: siteID,
+                synchronizeAllProductAttributeTerms(siteID: siteID,
```

Comment:
Super nit: Indentation fix needed.

Reply:
Done in 7920636dcc371e26917c35e6df9010e84f44d062.